modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -269,10 +271,36 @@ class NDB_Form_User_Accounts extends NDB_Form
             // END multi-site UPDATE
 
             // EXAMINER UPDATE
+            // get all fields that are related to examiners
+            $ex_curr_sites  = array();
+            $ex_prev_sites  = array();
+            $ex_radiologist = 'N';
+            $ex_pending     = 'Y';
+
+            foreach ($values as $k => $v) {
+                //examiner fields
+                if (preg_match("/^ex_[0-9]+$/", $k)) {
+                    //get centerID
+                    $parse_key = explode('_', $k);
+                    $cid       = $parse_key[1];
+                    $ex_curr_sites[$k] = $cid;
+                }
+                if ($k === 'examiner_radiologist') {
+                    $ex_radiologist = $v;
+                }
+                if ($k === 'examiner_pending') {
+                    $ex_pending = $v;
+                }
+            }
+
             // first check if an update is needed then actually do it
-            if (!empty($values['examiner_radiologist'])
-                && !empty($values['examiner_pending'])
+            if (!empty($ex_radiologist)
+                && !empty($ex_pending)
+                && !empty($ex_curr_sites)
             ) {
+                //modify examiner radiologist to be in the binary format 0-1
+                $ex_radiologist = $ex_radiologist === 'Y' ? 1:0;
```

Comment:
legacy: radiologist is saved as 1/0 not Y/N however php code manipulation of Y/N is safer

Reply:
wouldn't it be safer to have it as a boolean? Why 1/0?

Reply:
false is also empty, and most of the code checks for empty rather than checking for null and empty string and empty arrays seperately

and the 1/0 is the actual value in the DB that should be sent in the insert or update,